### PR TITLE
Add model for Christie Digital projectors

### PR DESCRIPTION
**Pull request by davromaniak**
Created a model for Christie Digital projectors, I only tested it on the only have I have, a CP2210.

I took inspiration from the epson-projector model, as it also monitors lamp life and it's what I needed.

Thanks.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/discovery/sensors/count/christie-projector.inc.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+/*
```
Comment (Jellyfrog) [inline includes/discovery/sensors/count/christie-projector.inc.php:2]: Any reason this is not in yaml?

Comment (davromaniak) [reply includes/discovery/sensors/count/christie-projector.inc.php:2]: Hi.

To be honest, I copied the epson-projector model and adapted to my needs.

I'll check this week or next week how to to it in YAML.

Thanks.

**Comment by davromaniak** [comment]
Hi.

I will continue working on this PR (switching from PHP code to YAML for the graph) during the second half of July, as I have limited access to the projector.

Thanks.

**Comment by murrant** [comment]
You'll need a MIB to use yaml.

**Comment by davromaniak** [comment]
Hi.

Well, the MIBs are restricted to customers only (and can't be redistributed), and even that, you'll only get the MIBs related to the devices you have support for. So... I can't do it with YAML and can't share the MIBs.

BTW, I still don't have access to the projectors as of today, if there's some fixes needed, I'll try next week.

Thanks.

**Comment by bayden10** [comment]
Hi, any progress with this as I have several Christie LHD720i and I'm able to SNMP to all devices but grabs the wrong OID, states it's Linux .1.3.6.1.4.1.8072.3.2.10 - I have the MIB file for these and installed to /mibs but missing the YAML.

Any thoughts or guidance would be appreciated.

**Comment by electrocret** [comment]
> Hi.
> 
> Well, the MIBs are restricted to customers only (and can't be redistributed), and even that, you'll only get the MIBs related to the devices you have support for. So... I can't do it with YAML and can't share the MIBs.
> 
> BTW, I still don't have access to the projectors as of today, if there's some fixes needed, I'll try next week.
> 
> Thanks.

If Christie Projector does not allow the redistribution of their MIBs, then that's a very poor business practice on their part. If you have a support contract with them, then you may want to reach out to them and ask for clarification on their distribution rules of their MIBs, and maybe explain why you're needing it to be distributable.

Not allowing the distribution of the MIB is like not allowing the distribution of the user manual for their equipment. The MIB is the instructions for the data their equipment makes available via SNMP. Without it, no monitoring system will be able to accurately support their equipment, because it won't have their instructions on how to support their equipment.

**Comment by bayden10** [comment]
You can access the Projector and download the MIB directly from the Web interface. Not sure that problem is. I'm just looking for help in getting this model added.

**Comment by murrant** [comment]
@bayden10 LibreNMS would need to be able to redistribute the MIB.

**Review by murrant — APPROVED** [review]

**Comment by bayden10** [comment]
I have spoken with Christie support and they state model LHD720i is no longer available for technical support. The manufacturer for this model was done by Hitachi. 

I can provide the MIB for this model as it's no longer available for support from the manufacturer. 


Header of MIB:

```
- --------------------------------------------------------
-- File: fpj2-ExMib.mib
--
-- Hitachi Maxell, Ltd.
--
-- Rev.1.1 2010.03.17
-- Rev.2.0 2010.07.27
-- Rev.2.1 2011.05.12
-- Rev.2.2 2011.10.19
-- Rev.2.3 2011.10.28
-- Rev.2.4 2011.11.09
-- Rev.2.5 2012.10.18
-- Rev.2.6 2013.05.27
-- Rev.2.7 2013.12.25
-- Rev.2.8 2014.02.13
-- Rev.2.9 2014.05.08
-- Rev.3.0 2014.05.13
-- Rev.3.1 2014.11.14
-- Rev.3.2 2015.02.13
-- Rev.3.3 2015.04.08

-- --------------------------------------------------------

FPJ2-EXMIB-MIB DEFINITIONS ::= BEGIN

	IMPORTS
		enterprises
			FROM RFC1155-SMI
		OBJECT-TYPE
			FROM RFC-1212;

	hitachi 	OBJECT IDENTIFIER ::= { enterprises 116 }

	system		OBJECT IDENTIFIER ::= { hitachi 3 }

	projector 	OBJECT IDENTIFIER ::= { system 37 }

	fpj2		OBJECT IDENTIFIER ::= { projector 2 }

	systemExMib	OBJECT IDENTIFIER ::= { hitachi 5 }

	projectorExMib	OBJECT IDENTIFIER ::= { systemExMib 37 }

	fpj2-ExMib	OBJECT IDENTIFIER ::= { projectorExMib 2 }


```

Let me know if you would like the MIB for this model. I have added custom OIDs to my monitoring but would be nice to have the full suite.

**Comment by bayden10** [comment]
Just wondering about the manufacturer directory for this MIB, should it be christie or hitachimaxell ? Once answered I will upload.

**Comment by murrant** [comment]
@bayden10 your choice.

**Pull request merged**